Documentation/TopLevelObjects/Plugin.rst

```diff
@@ -215,6 +215,34 @@ persistence.storagePid
              storagePid = 42
            }
          }
+.. _setup-plugin-persistence-recursive:
+
+persistence.recursive
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. container:: table-row
+
+   Property
+      persistence.recursive
+
+   Data type
+      :ref:`data-type-integer`
+
+   Description
+      **Only for Extbase plugins**. Number of how many sub-levels of the 
```

Comment:
```suggestion
**Only for Extbase plugins**. Number of sub-levels of the 
```

Reply:
@sarahmccarthy123 Please add the indent at the beginning again.

Reply:
```suggestion
      **Only for Extbase plugins**. Number of sub-levels of the 
```

Reply:
@sarahmccarthy123 I can't commit the suggestion, can you?